engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -23,6 +45,34 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryStatusAndRespond(id: WorkflowId): Unit = {
+    val sndr = sender()
+    dataAccess.getWorkflowStatus(id) onComplete {
```

Comment:
that's a good point. my best off the cuff thought is that we should be allowed to know if an id at least exists in the system, but that implies gleaning that knowledge from outside metadata